src/lib/content-utils.ts

```diff
@@ -404,10 +437,13 @@ class ContentUtils implements IContentUtils {
 		const result = nodes
 			.map((node) => this.processOneASTNode(node, parseContext))
 			.filter(Boolean);
-		this.updateLinks(result, parseContext);
+		result.forEach(r => this.updateLinksInNode(r, parseContext));
 		this.promoteInlines(result, parseContext);
 		//this.promoteFigures(result, parseContext);
```

Comment:
is `//this.promoteFigures(result, parseContext);` relevant to any future use cases?

Reply:
Removed